client/src/main/java/org/apache/cassandra/sidecar/client/RequestContext.java

```diff
@@ -486,6 +489,29 @@ public Builder connectedClientStatsRequest()
             return request(new ConnectedClientStatsRequest());
         }
 
+        /**
+         * Sets the {@code request} to be a {@link JobStatusRequest} and returns a reference to this Builder
+         * enabling method chaining.
+         *
+         * @return a reference to this Builder
+         */
+        public Builder jobStatusRequest(String jobId)
```

Comment:
Minor, to be in.line with listJobsRequest() naming style below, can we have this named statusJobRequest?